### DISCUSSION TITLE
Deno support

### DISCUSSION THREAD

**Discussion by hqnna** [ReactiveX/rxjs#5971] [Ideas / Feature request]
## Feature Request

Support Deno now that it has reached a stable api at version 1.0.0

**Describe the solution you'd like**

Add Deno as a supported runtime now that it has had an api freeze and has released it's first stable version.

**Additional context**

In an issue made in the past, it was suggested for Deno runtime support to be added, see: #4937. Deno has officially released version 1.0.0, and with this the internal api has been frozen and stabilized to a higher extent.

**Comment by kwonoj** [comment]
We will keep evaluate possibility, but at this moment supporting existing runtime better like https://github.com/ReactiveX/rxjs/issues/4416 have higher priority (which also makes step closer to deno support) but that alone require severe major breaking changes.

I recommend to use some of existing cjs interop lib from deno's userland, as this probably need time & consideration.

**Comment by garronej** [comment]
@xpyxel In the meantime you could give [EVT](https://evt.land) a try. It cannot compete featurewise yet but it got some good things going for it, first being to be fully cross-compatible Deno/Node/Browser/RN.

**Comment by benlesh** [comment]
I think we should discuss these issues at the next meeting. Deno is a run time with growing usage, and I think we should support it.

**Comment by garronej** [comment]
Hi @benlesh , if you want to support Deno without introducing breaking changes and without having to maintain a port you can use [Denoify](https://github.com/garronej/Denoify)
